diff --git a/src/Compositor.cpp b/src/Compositor.cpp
index 66e23b6..04202a0 100644
--- a/src/Compositor.cpp
+++ b/src/Compositor.cpp
@@ -939,7 +939,7 @@ void CCompositor::focusSurface(wlr_surface* pSurface, CWindow* pWindowOwner) {
         wlr_seat_keyboard_clear_focus(m_sSeat.seat);
         g_pEventManager->postEvent(SHyprIPCEvent{"activewindow", ","}); // unfocused
         g_pEventManager->postEvent(SHyprIPCEvent{"activewindowv2", ","});
-        EMIT_HOOK_EVENT("keyboardFocus", nullptr);
+        EMIT_HOOK_EVENT("keyboardFocus", (wlr_surface*)nullptr);
         m_pLastFocus = nullptr;
         return;
     }